packages/ns-api/files/ns.ovpnrw

```diff
@@ -882,6 +886,120 @@ def download_user_2fa(ovpninstance, username):
     else:
         return utils.validation_error("username", "2fa_download_failed", username)
 
+def _get_database(ovpninstance):
```

Comment:
Remove the whole function: it's not needed anymore

Reply:
done